src/taipy/core/config/data_node_config.py

```diff
@@ -120,6 +120,7 @@ class DataNodeConfig(Section):
     # SQL
     _REQUIRED_READ_QUERY_SQL_PROPERTY = "read_query"
     _REQUIRED_WRITE_QUERY_BUILDER_SQL_PROPERTY = "write_query_builder"
+    _OPTIONAL_APPEND_QUERY_BUILDER_SQL_PROPERTY = "append_query_builder"
```

Comment:
If it is optional, what happens when dn.append(data) is called? DO we have a default behavior? do we raise an error? Do we want to make it required?

Reply:
We definitely do not want to make this required. I should and will raise an error on `append()` method if the attribute is not provided.